bot/exts/info/help.py

```diff
@@ -264,7 +264,8 @@ async def subcommand_not_found(self, command: Command, string: str) -> "HelpQuer
 
     async def send_error_message(self, error: HelpQueryNotFound) -> None:
         """Send the error message to the channel."""
-        embed = Embed(colour=Colour.red(), title=str(error))
+        # 2257 the title can be too big. so we trim it
+        embed = Embed(colour=Colour.red(), title=str(error)[:256])
```

Comment:
a title of 256 characters is far too long. I know discord allows it to be this long, but that doesn't mean it would be a good title.
```suggestion
        embed = Embed(
            colour=Colour.red(),
            title=textwrap.shorten(error, width=150, placeholder="…"),
        )
```
Doing something like this, where we limit to 150 characters, and also append an ellipses would be better IMO.

This suggestion will also require an `import textwrap` at the top of the file.